packages/elements/src/pagination/index.ts

```diff
@@ -358,11 +364,31 @@ export class Pagination extends BasicElement {
     return this.useNextButton && !this.infinitePaginate;
   }
 
+  /**
+   * @override
+   */
+  protected update (changedProperties: PropertyValues): void {
+    super.update(changedProperties);
+    if (changedProperties.has('max')) {
+      if (!this.infinitePaginate) {
+        this.input.setAttribute('aria-valuemax', this.max);
+      }
+      else {
+        this.input.removeAttribute('aria-valuemax');
+      }
+    }
+  }
+
   /**
    * @override
    */
   protected updated (changedProperties: PropertyValues): void {
     super.updated(changedProperties);
+
+    if (changedProperties.has('value')) {
+      this.input.setAttribute('aria-valuenow', this.internalValue.toString());
+    }
```

Comment:
I'm a bit confused on `value` is check on `updated` and `max` is check on `update`, is there any difference at all ?

Reply:
It is not different; I moved all of them to the `updated` method.